pkg/pillar/utils/generics/lockmap_test.go

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) 2023 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package generics_test
+
+import (
+	"sync"
+	"testing"
+
+	"github.com/lf-edge/eve/pkg/pillar/utils/generics"
+	. "github.com/onsi/gomega"
+)
+
+func TestLockedMap(t *testing.T) {
+	t.Parallel()
+
+	g := NewGomegaWithT(t)
+
+	lm := generics.NewLockedMap[int, string]()
+
+	lm.Store(42, "bazinga")
+
+	got, ok := lm.Load(42)
+	g.Expect(ok).To(BeEquivalentTo(true))
+	g.Expect(got).To(BeEquivalentTo("bazinga"))
+
+	lm.Delete(42)
+	got, ok = lm.Load(42)
```

Comment:
Did you intend to g.Expect that got is nil?

Reply:
Actually it will be default initialized variable of value type of map in this case it should be empty string. Will update test and write comment just in case (because sometimes you look into tests to see how library (struct) was intended to use)

Reply:
Updated the test